CollAction/Controllers/AccountController.cs

```diff
@@ -253,18 +285,19 @@ public async Task<IActionResult> ForgotPassword(ForgotPasswordViewModel model)
                 var user = await _userManager.FindByEmailAsync(model.Email);
                 if (user == null)
                 {
-                    // Don't reveal that the user does not exist
```

Comment:
I'd prefer to keep this comment, stops people from adding validation

Reply:
Re-added